spec/controllers/proposals_controller_spec.rb

```diff
@@ -91,16 +91,22 @@
       end
     end
 
-    context 'details beta' do
+  end
+
+  describe '#show + new details' do
+    before do
```

Comment:
I think what @jessieay is after is something more like

```ruby
def setup_proposal_page
  login_as(user)
  proposal = create(:proposal, requester: user)
  get :show, id: proposal.id, detail: "new"
end
```

and then call `setup_proposal_page` within each `it` as necessary.